### PR TITLE
DOC-147/added ml adapters to data science

### PR DESCRIPTION
**Pull request by nerpaula**
previews:
- https://deploy-preview-1058--zealous-morse-14392b.netlify.app/docs/3.9/data-science-networkx-adapter.html
- https://deploy-preview-1058--zealous-morse-14392b.netlify.app/docs/3.9/data-science-dgl-adapter.html
- https://deploy-preview-1058--zealous-morse-14392b.netlify.app/docs/3.9/data-science-cugraph-adapter.html
- https://deploy-preview-1058--zealous-morse-14392b.netlify.app/docs/3.9/data-science-arango-rdf-adapter.html

Changes are applied to both 3.9 and 3.10

### CODE REVIEW

**Comment by cw00dw0rd** [comment]
Can we switch the name of the RDF adapter to `ArangoRDF` instead of `RDF Adapter`/`ArangoDB-RDF`?

**Comment by nerpaula** [comment]
> Can we switch the name of the RDF adapter to `ArangoRDF` instead of `RDF Adapter`/`ArangoDB-RDF`?

@cw00dw0rd done

File: 3.10/data-science-networkx-adapter.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+```bash
+pip install adbnx-adapter
+```
+
+## Quickstart
+
+The following examples show how to get started with ArangoDB-NetworkX Adapter.
+Check also the 
+[interactive tutorial](https://colab.research.google.com/github/arangoml/networkx-adapter/blob/master/examples/ArangoDB_NetworkX_Adapter.ipynb){:target="_blank"}.
```
[the remark this answers is not in the document]
Comment (aMahanna) [reply 3.10/data-science-networkx-adapter.md:35]: (would also need to apply this change to the 3.9 docs if we want to move forward with this)

[the remark this answers is not in the document]
Comment (nerpaula) [reply 3.10/data-science-networkx-adapter.md:35]: will apply all suggestions to both 3.9 and 3.10

**Review by cw00dw0rd — APPROVED** [review]
This looks great, thank you!

**Comment by cw00dw0rd** [comment]
@aMahanna can you also confirm

**Pull request merged**